cmd/monaco/integrationtest/account/utils_test.go

```diff
@@ -23,49 +23,10 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/files"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/require"
-	"regexp"
 	"strings"
 	"testing"
 )
 
-type monacoCmd struct {
-	cmd string
-	fs  afero.Fs
-}
-
-func monaco(cmd string) *monacoCmd {
-	cmd = regexp.MustCompile(`\s+`).ReplaceAllString(cmd, " ")
-	cmd = strings.Trim(cmd, " ")
-	cmd = strings.TrimPrefix(cmd, "monaco ")
-
-	return &monacoCmd{cmd: cmd}
-}
-func monacof(cmd string, args ...any) *monacoCmd {
-	return monaco(fmt.Sprintf(cmd, args...))
-}
-
-func (cmd *monacoCmd) withFs(fs afero.Fs) *monacoCmd {
```

Comment:
Do we ever need a `monacoCmd` without a `fs`?

Reply:
In some cases, the output of one command is input to another. As the output of monaco is always on disk, it is necessary to chain the

Reply:
I change the implementation to a single step command. A separate command is available in case FS needs to be provided.